kickstarter/android-oss
# NTV-182: missing values v1 Currency + No Reward

# 📲 What

- We exchange on `ProjectViewModel.kt` all the fetchProject call from V1 for the ones to GraphQL

# 🤔 Why

- New data will be needed on Project page only available via GraphQL

# 🛠 How

- Exchange network calls to V1 over the GraphQL ones

# 👀 See
- Same user experience as begore, exept for some Reward fields missing on the reward carousel, next Pr will be open for that.
- ⚠️  missing some fields on rewards, next PR will solve that part

https://user-images.githubusercontent.com/4083656/133351464-739c3412-0bca-40a6-aa74-301bfb6c421d.mp4


|  |  |

# 📋 QA
- Load project without log in, try to like a project, it will pressent the login screen, log in and make sure the project is liked.
- Like a project when logged in
- Make sure the currency displayed for your user has not changed compare to previous app version
- Back a project 
- Update pledge & payment method & cancel pledge & change reward flows working properly

# Story 📖

[NTV-182](Thttps://kickstarter.atlassian.net/browse/NTV-182)
